test/helpers/dummy-server.js

```diff
@@ -7,6 +7,27 @@ const startExpressAppPromise = (app, port) => {
     });
 };
 
+const parseHeaders = (headers, obj = {}) => {
```

Comment:
`headers` are `rawHeaders` right? Would be nice to name the variable this way or have the doc comment that it expects an array.